src/auryn/SyncBuffer.cpp

```diff
@@ -71,6 +71,7 @@ void SyncBuffer::init()
 	syncCount = 0;
 
 	max_send_size = 4;
+	send_bufx.resize(max_send_size);  // ALa added
```

Comment:
I have commited a draft version of a new SyncBuffer which is currently undergoing testing https://github.com/fzenke/auryn/tree/dev-syncbuffer - this version doesn't declare a second send_bufx array. Maybe you can check it out and tell me whether it works for you. If my suggested implementation works with your setup I would prefer it over the one in which an additional send_bufx is declared.